Library/Formula/rebar3.rb

```diff
@@ -0,0 +1,19 @@
+class Rebar3 < Formula
+  desc "Erlang build tool"
+  homepage "https://github.com/rebar/rebar3"
+  url "https://github.com/rebar/rebar3/archive/3.0.0-beta.4.tar.gz"
```

Comment:
So should I fork dev-only and move this formula there ? When the final release is done who moves it to homebrew ?